website/cue/reference/components/sources/http_scrape.cue

```diff
@@ -0,0 +1,216 @@
+package metadata
+
+components: sources: http_scrape: {
+	title: "HTTP Scrape"
+	alias: "http_scrape"
+
+	classes: {
+		commonly_used: false
+		delivery:      "at_least_once"
+		deployment_roles: ["daemon", "sidecar"]
+		development:   "beta"
+		egress_method: "batch"
+		stateful:      false
+	}
+
+	features: {
+		acknowledgements: false
+		codecs: {
+			enabled:         true
+			default_framing: "`bytes`"
+		}
+		collect: {
+			checkpoint: enabled: false
+			from: {
+				service: services.http_scrape
+
+				interface: socket: {
+					direction: "outgoing"
+					protocols: ["http"]
+					ssl: "optional"
+				}
+			}
+			proxy: enabled: true
+			tls: {
+				enabled:                true
+				can_verify_certificate: true
+				can_verify_hostname:    true
+				enabled_default:        false
+			}
+		}
+		multiline: enabled: false
+	}
+
+	support: {
+		requirements: []
+		warnings: []
+		notices: []
+	}
+
+	installation: {
+		platform_name: null
+	}
+
+	configuration: {
+		auth: configuration._http_auth & {_args: {
+			password_example: "${PASSWORD}"
+			username_example: "${USERNAME}"
+		}}
+		endpoint: {
+			description: "Endpoint to scrape observability data from."
+			required:    true
+			warnings: ["You must explicitly add the path to your endpoint."]
+			type: string: {
+				examples: ["http://127.0.0.1:9898/logs"]
+			}
+		}
+		headers: {
+			common:      false
+			description: "A list of HTTP headers to include in request."
+			required:    false
+			type: object: {
+				examples: [{"Your-Custom-Header": "it's-value"}]
+			}
+		}
+		query: {
+			common: false
+			description: """
+				Custom parameters for the scrape request query string.
+				One or more values for the same parameter key can be provided.
+				The parameters provided in this option are appended to the `endpoint` option.
+				"""
+			required: false
+			type: object: {
+				examples: [{"match[]": [#"{job="somejob"}"#, #"{__name__=~"job:.*"}"#]}]
```

Comment:
I think these examples could be better. They will likely be confusing to anyone not familiar with Prometheus.

Reply:
Yeah that was a shameful copy paste to get something there... I'll improve that 👍

Reply:
Attempted to address in https://github.com/vectordotdev/vector/pull/13793/commits/bb43aa7ea4ce7ee0f1edb07c64682b646dcaa384